src/OrchardCore.Modules/OrchardCore.Contents/Views/Contents-Button-Edit.SummaryAdmin.cshtml

```diff
@@ -1,12 +1,23 @@
+@using Microsoft.AspNetCore.Authorization
 @using OrchardCore.ContentManagement
+@inject IAuthorizationService AuthorizationService
 @inject IContentManager ContentManager
 
 @{
     ContentItem contentItem = Model.ContentItem;
     var hasPublished = await ContentManager.HasPublishedVersionAsync(contentItem);
+    var hasEditContentPermission = await AuthorizationService.AuthorizeAsync(User, OrchardCore.Contents.Permissions.EditContent, contentItem);
 }
 
-<a edit-for="@contentItem" asp-route-returnUrl="@FullRequestPath" class="btn btn-primary btn-sm">@T["Edit"]</a>
+@if(hasEditContentPermission)
+{
+    <a edit-for="@contentItem" asp-route-returnUrl="@FullRequestPath" class="btn btn-primary btn-sm">@T["Edit"]</a>
+}
+
+@if (hasPublished)
+{
+    <a display-for="@contentItem" class="btn btn-success btn-sm" itemprop="">@T["View"]</a>
+}
```

Comment:
<img width="990" alt="Screenshot 2020-08-20 at 09 51 14" src="https://user-images.githubusercontent.com/13782679/90748932-b7043980-e2ca-11ea-9b07-577db0bc8e4b.png">

???

Reply:
Strange!! let me check again ...

Reply:
Nah!! the condition get copied twice during merge conflict